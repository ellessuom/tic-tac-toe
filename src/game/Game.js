import React from 'react';
import Layout from './layout';
import Dashboard from './dashboard';
import Players from './players';

function Game() {
    return (
      <Layout>
        <Players />
        <Dashboard />
      </Layout>
    );
}


export default Game;
