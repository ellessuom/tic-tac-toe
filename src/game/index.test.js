import React from 'react';
import { shallow } from 'enzyme';
import Game from './index';

describe('<Game />', () => {
  it('should render app structure correctly', () => {
    const component = shallow(<Game />);

    expect(component).toMatchSnapshot();
  });
});
