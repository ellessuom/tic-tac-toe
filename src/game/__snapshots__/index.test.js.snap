// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Game /> should render app structure correctly 1`] = `
<DataProvider>
  <Layout>
    <Players />
    <Dashboard />
  </Layout>
</DataProvider>
`;
